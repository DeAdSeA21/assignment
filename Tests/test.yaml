dueDate: 2025-04-28
latePenalties: [10, 20, 100]
test:
  launch: ./loan
  timelimit: 10
  grading:
    - oracle: smart
      scoring: ByLine
      ws: true
      cap: 100
    - oracle: smart
      scoring: ByLine
      numbersOnly: true
      cap: 85
build:
  instructorFiles:
    exclude:
      - loan.cpp
  studentFiles:
    exclude:
      - makefile
  command: make -k
