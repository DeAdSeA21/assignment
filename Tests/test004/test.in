1000.00
0.01
4.0
