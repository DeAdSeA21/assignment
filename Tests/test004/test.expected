Starting loan balance: Annual interest rate: Monthly payment: 
Month	Balance
1	996.83
2	993.66
3	990.49
4	987.32
5	984.14
6	980.96
7	977.78
8	974.59
9	971.40
10	968.21
11	965.02
12	961.83
13	958.63
14	955.43
15	952.22
16	949.02
17	945.81
18	942.59
19	939.38
20	936.16
21	932.94
22	929.72
23	926.50
24	923.27
25	920.04
26	916.80
27	913.57
28	910.33
29	907.09
30	903.84
31	900.60
32	897.35
33	894.09
34	890.84
35	887.58
36	884.32
37	881.06
38	877.79
39	874.52
40	871.25
41	867.98
42	864.70
43	861.42
44	858.14
45	854.86
46	851.57
47	848.28
48	844.98
49	841.69
50	838.39
51	835.09
52	831.79
53	828.48
54	825.17
55	821.86
56	818.54
57	815.22
58	811.90
59	808.58
60	805.25
